docs/logging.py

```diff
@@ -0,0 +1,15 @@
+class Logger:
```

Comment:
Since this is already here I think it's fine to use it.

If we ant to use colors, here's a stack-overflow answer that shows how to configure a built in logger formatter so you can log different levels with different colors.